apps/site/assets/ts/stop/components/DeparturesAndMap.tsx

```diff
@@ -25,6 +25,13 @@ interface DeparturesAndMapProps {
   alerts: Alert[];
 }
 
+type DepartureFilter = {
+  route: Route;
+  directionId: DirectionId;
+  headsign: string;
+};
+export type DepartureFilterFn = (filters: DepartureFilter) => void;
```

Comment:
nitpick: Should this be defined in a helper file?

Reply:
I'd say it should stay close to the departure filtering functionality it's related to (the state, the set departure filters function, the unset departure filters function, etc). If it goes in a helper file, all the other related bits should join it.